valgrind_suppressions.txt

```diff
@@ -54,8 +54,6 @@
    match-leak-kinds: definite
    fun:*
    obj:*/plugins/platforms/libqoffscreen.so
-   obj:*/plugins/platforms/libqoffscreen.so
```

Comment:
i think there should be a `...` here on it's own line (after the`obj:*/plugins/platforms/libqoffscreen.so` and before the `fun:*QGuiApplicationPrivate*createPlatformIntegration*` ) as we don't know how many fun/obj's there will be as this seem to change depending on the system.

Reply:
Oh, I didn't realize the `...` you posted in the above comment was meant literally.